pkgs/by-name/pa/pappl2/package.nix

```diff
@@ -0,0 +1,61 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, libcups3
+, libjpeg
+, libpng
+, libusb1
+, pam
+, pkg-config
+, enableJpeg ? true
+, enablePng ? true
+, enableUsb ? true
+, enablePam ? true
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pappl2";
+  version = "unstable-2024-05-22";
```

Comment:
```suggestion
  version = "0-unstable-2024-05-22";
```

Reply:
Where is the precedent for this? Every package in nixpkgs I can find that doesn't have a stable version uses `unstable-<date>` as the version.

Reply:
1. Each and every one? A simple ripgrep on the codebase returns 224 matches for "0-unstable". 
   - OK, there are 1400 matches for "unstable". But the Nixpkgs codebase is hard to update...
3. The "precedent" is in the line 355 of [pkgs/README.md](https://github.com/NixOS/nixpkgs/blob/a0564731be68aa3aa461cb80924e8123c42a4d4f/pkgs/README.md?plain=1#L355-L357):

> If a project has no suitable preceding releases - e.g., no versions at all, or an incompatible versioning / tagging schema - then the latest upstream version in the above schema should be `0`.
> Example: Given a project that has no tags / released versions at all, or applies versionless tags like `latest` or `YYYY-MM-DD-Build`, a commit authored on March 15, 2022 would have `version = "0-unstable-2022-03-15"`.

Reply:
Fair enough. I'll use that, then.